mirror_mgmt/main.py

```diff
@@ -70,7 +73,7 @@ def main() -> None:
         create_mirrors()
     elif args.action == 'update_mirrors':
         validate()
-        update_mirrors()
+        update_mirrors([m.strip() for m in args.update_mirrors.split(',')] if args.update_mirrors else [])
```

Comment:
You can specify `type=` in `add_argument` to parse this value in-place, and also specify `default=[]`